src/pages/home.html

```diff
@@ -36,6 +36,335 @@
                 <li class="dot active w-2 h-2 rounded-full bg-white mr-3 transition-[1s]"></li>
             </ul>
         </div>
+
+        <div class="outstanding-products pt-[30px]">
+            <div class="container">
+                <h2
+                    class="text-main font-bold text-[17px] pt-[10px] pb-[18px] border-b border-b-[#ebebeb] border-solid mb-[20px]">
+                    <a href="#" class="pb-[15px] border-b-[3px] border-b-main border-solid"><span data-i18n="title">Sản phẩm nổi bật</span></a>
```

Comment:
```suggestion
                    <a href="#" class="pb-[15px] border-b-[3px] border-b-main border-solid"><span data-i18n="title"></span></a>
```
Chỗ này đã có i18n rồi, e thực hiện xóa các cái text bên trong này đi nha